<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetBaseStation</class>
 <widget class="QWidget" name="WidgetBaseStation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>939</width>
    <height>758</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="5,1,10">
   <item>
    <widget class="QGroupBox" name="groupBoxMessage">
     <property name="title">
      <string>工位基础信息</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0" rowspan="2">
       <widget class="QLabel" name="labelStationCode">
        <property name="text">
         <string>工位代号：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="labelStationType">
        <property name="text">
         <string>工位类型：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelDescription">
        <property name="text">
         <string>备注：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="editStationCode"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelStationID">
        <property name="text">
         <string>工位编号：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="labelStationName">
        <property name="text">
         <string>工位名称：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QComboBox" name="comboBoxStationType"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="editStationID"/>
      </item>
      <item row="3" column="1" colspan="3">
       <widget class="QTextEdit" name="editDescription"/>
      </item>
      <item row="0" column="3" rowspan="2">
       <widget class="QLineEdit" name="editStationName"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ButtonGroup" name="buttonGroup" native="true"/>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>工位列表</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableView" name="tableView"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ButtonGroup</class>
   <extends>QWidget</extends>
   <header>ButtonGroup.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
